File: snippets/fsharp/System/String/IndexOfAny/IndexOfAny1.fs
module IndexOfAny1.fs
// <Snippet1>
let chars = [| 'a'; 'e'; 'i'; 'o'; 'u'; 'y'
               'A'; 'E'; 'I'; 'O'; 'U'; 'Y' |]
let s = "The long and winding road..."
printfn $"The first vowel in \n   {s}\nis found at index {s.IndexOfAny chars}"

// The example displays the following output:
//       The first vowel in
//          The long and winding road...
//       is found at index 2
// </Snippet1>

File: snippets/csharp/System/String/IndexOfAny/IndexOfAny1.cs
using System;

public class Example1
{
   public static void Run()
   {
      // <Snippet1>
      char[] chars = { 'a', 'e', 'i', 'o', 'u', 'y',
                       'A', 'E', 'I', 'O', 'U', 'Y' };
      String s = "The long and winding road...";
      Console.WriteLine($"""
         The first vowel in
         '{s}'
         is found at index {s.IndexOfAny(chars)}
         """);

      // The example displays the following output:
      //       The first vowel in
      //       'The long and winding road...'
      //       is found at index 2
      // </Snippet1>
   }
}
